tests/lifecycle_wait_test.go

```diff
@@ -181,13 +183,10 @@ func TestLifecycleWaitForNavigation(t *testing.T) {
 				click := func() error {
 					return p.Click(`a`, nil)
 				}
-				runAll := k6ext.Promise(tb.ctx, func() (result any, reason error) {
-					ctx, cancel := context.WithTimeout(tb.ctx, 5*time.Second)
-					defer cancel()
-					return nil, tb.run(ctx, waitForNav, click)
-				})
 
-				return tb.promise(runAll)
+				ctx, cancel := context.WithTimeout(tb.ctx, 5*time.Second)
```

Comment:
5 seconds or 1 sec as the previous suggestion?

Reply:
The `timeout` test runs an `asserFunc` instead of running the main body of the test, so it completes before reaching the 5 second timeout step.

Reply:
Oh, it's good then 👍